packages/trader/src/AppV2/Components/BottomNav/bottom-nav.tsx

```diff
@@ -83,8 +96,8 @@ const BottomNav = observer(({ children, className, onScroll }: BottomNavProps) =
                         <Navigation.BottomAction
                             key={index}
                             index={index}
-                            activeIcon={item.icon}
-                            icon={item.icon}
+                            activeIcon={<></>}
+                            icon={index === selectedIndex ? item.activeIcon : item.icon}
```

Comment:
In Quill there is a part of code
```
                {(icon || activeIcon) && (
                    <span className="quill-navigation-bottom-bar__action-icon">
                        {selected ? activeIcon : icon}
                    </span>
                )}
```
It should insert icon or active icon based on if it's selected.
Will this work?
```
                  activeIcon={item.activeIcon }
                  icon={ item.icon}
```

Reply:
no that didnt work. thats why i changed the logic to what i have